pkg/usermanagement/client.go

```diff
@@ -980,6 +1002,10 @@ func (c *Client) GetAuthorizationURL(opts GetAuthorizationURLOpts) (*url.URL, er
 		query.Set("screen_hint", string(opts.ScreenHint))
 	}
 
+	if len(opts.ProviderScopes) > 0 {
+		query.Set("provider_scopes", strings.Join(opts.ProviderScopes, ","))
```

Comment:
Should these be space-separated? Similar to [the PHP SDK implementation](https://github.com/workos/workos-php/pull/289/files#diff-e10143672057f6945a897ea4fe63716e89c429029f0bb2a0e3414955e6081a31R695).

```suggestion
		query.Set("provider_scopes", strings.Join(opts.ProviderScopes, " "))
```

Reply:
Ah it is actually the PHP SDK code that is wrong - our API is expecting a comma-separated list. I'll update my PHP PR.